lib/puppet/provider/package/ports.rb

```diff
@@ -6,9 +6,6 @@
     :portuninstall => "/usr/local/sbin/pkg_deinstall",
     :portinfo => "/usr/sbin/pkg_info"
 
-  defaultfor :operatingsystem => :freebsd
```

Comment:
It looks like this was addressed in commit 69d9ba6b4f66c77dcbff0456d48f36059d35d1b2